### ISSUE TITLE
New submissions for Thu, 17 Aug 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#83]
# New submissions for Thu, 17 Aug 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### Interaction of H$_2$S with H atoms on grain surfaces under molecular  cloud conditions
 - **Authors:** Julia C. Santos, Harold Linnartz, Ko-Ju Chuang
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2308.07960
 - **Pdf link:** https://arxiv.org/pdf/2308.07960
 - **Abstract**
 Hydrogen sulfide (H$_2$S) is thought to be efficiently formed on grain surfaces through the successive hydrogenation of S atoms. Its non-detection so far in astronomical observations of icy dust mantles thus indicates that effective destruction pathways must play a significant role in its interstellar abundance. While chemical desorption has been shown to remove H$_2$S very efficiently from the ice, in line with H$_2$S gas-phase detections, possible solid-state chemistry triggered by the related HS radical have been largely disregarded so far -- despite it being an essential intermediate in the H$_2$S + H reaction scheme. We aim to thoroughly investigate the fate of H$_2$S upon H-atom impact under molecular cloud conditions, providing a comprehensive analysis combined with detailed quantification of both the chemical desorption and ice chemistry that ensues. Experiments are performed in an ultrahigh vacuum chamber at temperatures between 10--16 K. The changes in the solid phase during H-atom bombardment are monitored in situ by means of reflection absorption infrared spectroscopy (RAIRS), and desorbed species are measured with a quadrupole mass spectrometer (QMS). We confirm the formation of H$_2$S$_2$ via reactions involving H$_2$S + H, and quantify its formation cross section under the employed experimental conditions. Additionally, we directly assess the chemical desorption of H$_2$S by measuring the gas-phase desorption signals with the QMS, providing unambiguous desorption cross sections. Chemical desorption of H$_2$S$_2$ was not observed. The relative decrease of H$_2$S ices by chemical desorption changes from ~85% to ~74% between temperatures of 10 and 16 K, while the decrease as the result of H$_2$S$_2$ formation is enhanced from ~5% to ~26%, suggesting an increasingly relevant sulfur chemistry induced by HS radicals at warmer environments. The astronomical implications are further discussed.
#### Exploring asymmetric substructures of the outer disk based on the  conjugate angle of the radial action
 - **Authors:** Y. Xu, C. Liu, Z. Li, H. Tian, Sarah A. Bird, H. J. Newberg, S. Shao, L. C. Deng
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2308.08092
 - **Pdf link:** https://arxiv.org/pdf/2308.08092
 - **Abstract**
 We use the conjugate angle of radial action ($\theta_R$), the best representation of the orbital phase, to explore the "mid-plane, north branch, south branch" and "Monoceros area" disk structures that were previously revealed in the LAMOST K giants (Xu et al. 2020). The former three substructures, identified by their 3D kinematical distributions, have been shown to be projections of the phase space spiral (resulting from nonequilibrium phase mixing). In this work, we find that all of these substructures associated with the phase spiral show high aggregation in conjugate angle phase space, indicating that the clumping in conjugate angle space is a feature of ongoing, incomplete phase mixing. We do not find the $Z-V_Z$ phase spiral located in the "Monoceros area", but we do find a very highly concentrated substructure in the quadrant of conjugate angle space with the orbital phase from the apocenter to the guiding radius. The existence of the clump in conjugate angle space provides a complementary way to connect the "Monoceros area" with the direct response to a perturbation from a significant gravitationally interactive event. Using test particle simulations, we show that these features are analogous to disturbances caused by the impact of the last passage of the Sagittarius dwarf spheroidal galaxy.
#### Slow diffusion around pulsar $γ$-ray halos and its impact on cosmic  rays propagation
 - **Authors:** Xiao-Jun Bi
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); High Energy Physics - Phenomenology (hep-ph)
 - **Arxiv link:** https://arxiv.org/abs/2308.08099
 - **Pdf link:** https://arxiv.org/pdf/2308.08099
 - **Abstract**
 The diffusion coefficients around the pulsar $\gamma$-ray halos are highly suppressed compared with the value in the interstellar medium. It is suggested in the literature that the $\gamma$-ray halos can be explained by a ballistic-diffusive (BD) propagation without slow diffusion. However our calculation shows that the BD propagation can not account for the $\gamma$-ray halo profile well. Furthermore the transfer efficiency of the pulsar spin down energy to the high energy electrons and positrons is even larger than 1 in the BD scenario. Therefore slow diffusion is necessary to account for the pulsar $\gamma$-ray halos. Taking the slow diffusion into account the contribution of positron flux originated from nearby pulsars to the AMS-02 data is reexamined. We may also expect a slow diffusion disk of the Milky Way as many such slow diffusion regions exist. The positron contribution to the AMS-02 data from dark matter annihilation in the new propagation model is also reexamined. We find that the dark matter scenario satisfies all the $\gamma$-ray limits in the new propagation model.
#### Galactic and Extragalactic Analysis of the Astrophysical Muon Neutrino  Flux with 12.3 years of IceCube Track Data
 - **Authors:** Philipp Fürst (for the IceCube Collaboration)
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2308.08233
 - **Pdf link:** https://arxiv.org/pdf/2308.08233
 - **Abstract**
 The Ice Cube Neutrino Observatory has been measuring an isotropic astrophysical neutrino flux in multiple detection channels for almost a decade. Galactic diffuse emission, which arises from the interactions between cosmic rays and the interstellar medium, is an expected signal in IceCube. The superposition of an extragalactic flux and a galactic flux results in directional structure and variations in the spectrum. In this work, we use 12.3 years of high-purity muon-neutrino induced muon track data to perform a dedicated search for this galactic emission, combined with a spectral measurement of the isotropic astrophysical neutrino flux. To distinguish a galactic component from the dominant atmospheric and isotropic astrophysical components, the precise directional information available for muon tracks is fully utilized in a three-dimensional forward folding likelihood fit. We test a state-of-the-art model prediction of galactic diffuse emission based on recent cosmic ray data (CRINGE). We fit this prediction as a template scaled by a factor $\Psi_{\mathrm{CRINGE}}$, and find $2.9\pm 1.1 \times \Psi_{\mathrm{CRINGE}}$ with a significance of $2.7\sigma$ in an energy range between 400 GeV and 60 TeV in the Northern Sky.
#### Neutrinos and Asteroseismology of Stars over the Helium Flash
 - **Authors:** Diogo Capelo, Ilídio Lopes
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2308.08282
 - **Pdf link:** https://arxiv.org/pdf/2308.08282
 - **Abstract**
 The helium flash, occurring in stars of 0.6-2.0 M$_\odot$ at the end of the red giant branch, is not observable via optical means due to the energy of the process being used to lift the core out of degeneracy. Neutrinos, which are linked to the ignition of reactions triggered during the flash and serve as the only cooling process in the inert core, can help characterize changes in internal structure. In this work, we create 18 stellar models across three mass and six metallicity values, chosen in the context of the stellar abundance problem, to compare the evolutionary path up to and probe the helium flash by conducting a detailed study of neutrino emission throughout this crucial phase of stellar evolution. We demonstrate how thermal neutrino emissions could have an imprint on global asteroseismic parameters and use them as an additional tool to infer the impact of compositional changes. We find that a precision of 0.3 $\mu$Hz in the determination of $\Delta \nu$ is enough to distinguish between between the two most prominent solar composition models and confirm that asteroseismic observation can be enough to classify a star as undergoing the process of helium subflashes. We also predict nuclear neutrino emission fluxes and their evolution for all relevant sources.
#### Infrared spectra of TiO2 clusters for hot Jupiter atmospheres
 - **Authors:** J.P. Sindel, Ch. Helling, D. Gobrecht, K.L. Chubb, L. Decin
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2308.08297
 - **Pdf link:** https://arxiv.org/pdf/2308.08297
 - **Abstract**
 Context. Clouds seem unavoidable in cool and dense environments, and hence, are necessary to explain observations of exoplanet atmospheres, most recently of WASP 96b with JWST. Understanding the formation of cloud condensation nuclei in non-terrestrial environments is therefore crucial to develop accurate models to interpret present and future observations. Aims. The goal of the paper is to support observations with infrared spectra for (TiO2)N clusters in order to study cloud formation in exoplanet atmospheres. Methods. Vibrational frequencies are derived from quantum-chemical calculations for 123 (TiO2)-clusters and their isomers, and line-broadening mechanisms are evaluated. Cluster spectra are calculated for several atmospheric levels for two example exoplanet atmospheres (WASP 121b-like and WASP 96b-like) to identify possible spectral fingerprints for cloud formation. Results. Rotational motion of and transitions in the clusters cause significant line broadening, so that individual vibrational lines are broadened beyond the spectral resolution of the medium resolution mode of the JWST mid-infrared instrument MIRI at R = 3000. However, each individual cluster isomer exhibits a "fingerprint" IR spectrum. In particular, larger (TiO2)-clusters have distinctly different spectra from smaller clusters. Morning and evening terminator for the same planet can exhibit different total absorbances due to different cluster sizes being more abundant. Conclusions. The largest (TiO2)-clusters are not necessarily the most abundant (TiO2)-clusters in the high-altitude regions of ultra-hot Jupiters, and the different cluster isomers will contribute to the local absorbance. Planets with a considerable day-night asymmetry will be most suitable to search for (TiO2)-cluster isomers in order to improve cloud formation modelling.
#### Fallback onto Kicked Neutron Stars and its Effect on Spin-Kick Alignment
 - **Authors:** B. Müller (Monash University)
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2308.08312
 - **Pdf link:** https://arxiv.org/pdf/2308.08312
 - **Abstract**
 Fallback in core-collapse supernova explosions is potentially of significant importance for the birth spins of neutron stars and black holes. It has recently been pointed out that the angular momentum imparted onto a compact remnant by fallback material is subtly intertwined with its kick because fallback onto a moving neutron star or black hole will preferentially come for a conical region around its direction of travel. We show that contrary to earlier expectations such one-sided fallback accretion onto a neutron star will tend to produce spin-kick misalignment. Since the baroclinic driving term in the vorticity equation is perpendicular to the nearly radial pressure gradient, convective eddies in the progenitor as well as Rayleigh-Taylor plumes growing during the explosion primarily carry angular momentum perpendicular to the radial direction. Fallback material from the accretion volume of a moving neutron star therefore carries substantial angular momentum perpendicular to the kick velocity. We estimate the seed angular momentum fluctuations from convective motions in core-collapse supernova progenitors and argue that accreted fallback material will almost invariably be accreted with the maximum permissible specific angular momentum for reaching the Alfv\'en radius. This imposes a limit of $\mathord{\sim}10^{-2}M_\odot$ of fallback accretion for fast-spinning young neutron stars with periods of $\mathord{\sim}20\,\mathrm{ms}$ and less for longer birth spin periods.
#### Galaxy Rotation Curve Fitting Using Machine Learning Tools
 - **Authors:** Carlos R. Argüelles, Santiago Collazo
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); General Relativity and Quantum Cosmology (gr-qc)
 - **Arxiv link:** https://arxiv.org/abs/2308.08420
 - **Pdf link:** https://arxiv.org/pdf/2308.08420
 - **Abstract**
 Galaxy rotation curve (RC) fitting is an important technique which allows the placement of constraints on different kinds of dark matter (DM) halo models. In the case of non-phenomenological DM profiles with no analytic expressions, the art of finding RC best-fits including the full baryonic $+$ DM free parameters can be difficult and time-consuming. In the present work, we use a gradient descent method used in the backpropagation process of training a neural network, to fit the so-called Grand Rotation Curve of the Milky Way (MW) ranging from $\sim$1 pc all the way to $\sim$$10^5$ pc. We model the mass distribution of our Galaxy including a bulge (inner $+$ main), a disk, and a fermionic dark matter (DM) halo known as the Ruffini-Arg\"uelles-Rueda (RAR) model. This is a semi-analytical model built from first-principle physics such as (quantum) statistical mechanics and thermodynamics, whose more general density profile has a dense core -- diluted halo morphology with no analytic expression. As shown recently and further verified here, the dark and compact fermion-core can work as an alternative to the central black hole in SgrA* when including data at milliparsec scales from the S-cluster stars. Thus, we show the ability of this state-of-the-art machine learning tool in providing the best-fit parameters to the overall MW RC in the $10^{-2}$--$10^5$ pc range, in a few hours of CPU time.
#### Gravitational Wave Emission from a Cosmic String Loop, I: Global Case
 - **Authors:** Jorge Baeza-Ballesteros, Edmund J. Copeland, Daniel G. Figueroa, Joanes Lizarraga
 - **Subjects:** Cosmology and Nongalactic Astrophysics (astro-ph.CO); High Energy Physics - Phenomenology (hep-ph)
 - **Arxiv link:** https://arxiv.org/abs/2308.08456
 - **Pdf link:** https://arxiv.org/pdf/2308.08456
 - **Abstract**
 We study the simultaneous decay of global string loops into scalar particles (massless and massive modes) and gravitational waves (GWs). Using field theory simulations in flat space-time of %isolated loops with initial length $\sim 80-1700$ times their core width, we determine the power emitted by a loop into scalar particles, $P_{\varphi}$, and GWs, $P_{\rm GW}$, and characterize the loop-decay timescale as a function of its initial length, energy and angular momentum. We quantify infrared and ultraviolet lattice dependencies of our results. For all type of loops and initial conditions considered, GW emission is always suppressed compared to particles as $P_{\rm GW}/P_{\varphi} \approx \mathcal{O}(10)(v/m_\text{p})^2\ll 1$, where $v$ is the vacuum expectation value associated with string formation. Our results suggest that the GW background from a global string network, such as in dark matter axion scenarios, will be highly suppressed.

by olozhika (Xing Yuchen). 

2023-08-17
